examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccounts.agent.ts

```diff
@@ -5,11 +5,32 @@ import { NOT_FOUND_ERR_MESSAGE } from '../api/err.messages';
 import { BpiSubjectAccountStorageAgent } from './bpiSubjectAccountsStorage.agent';
 import { BpiSubjectAccount } from '../models/bpiSubjectAccount';
 import { BpiSubject } from '../../bpiSubjects/models/bpiSubject';
+import { BpiSubjectStorageAgent } from '../../bpiSubjects/agents/bpiSubjectsStorage.agent';
 
 // Agent methods have extremely declarative names and perform a single task
 @Injectable()
 export class BpiSubjectAccountAgent {
-  constructor(private storageAgent: BpiSubjectAccountStorageAgent) {}
+  constructor(
+    private storageAgent: BpiSubjectAccountStorageAgent,
+    private storageSubjectAgent: BpiSubjectStorageAgent,
```

Comment:
subjectAccountStorageAgent

Reply:
no this is subject agent, to be able to query creator and owner subject by id to check if they exist

Reply:
ah ok i understand now, comments for this and line below seem reversed to me